piker/ui/_style.py

```diff
@@ -99,6 +113,14 @@ def configure_to_dpi(self, screen: QtGui.QScreen | None = None):
         listed in the script in ``snippets/qt_screen_info.py``.
 
         '''
+        if self._font_size is not None:
```

Comment:
hmm maybe it's handy to eventually add a `.reset()` meth as well in case the window gets moved to another display and we want to re-compute the dynamic-by-DPI font size?

(will stick in follow up issue as per above comments)